src/resources/account/account.router.js

```diff
@@ -0,0 +1,21 @@
+const { Router } = require('express');
+const {
+  createAccount, deleteAccount, getAccountById, updateAccount,
+} = require('./account.service');
+
+const router = Router();
+// eslint-disable-next-line no-unused-vars
+const httpError = require('../../tools/httpError');
+// eslint-disable-next-line no-empty-pattern
+
+router.get('/:id', getAccountById);
+
+router.post('/', createAccount);
+
+// update
```

Comment:
no need to add comments in this place. the logic very simple and that is why no need to add some comments here. only if you have some difficult logic then please add some comments

Reply:
I got you